README.md

```diff
@@ -83,6 +93,11 @@ Start the SDK instance to observe transactions. </br>
 
 ### <a id="stop"> Stop Observing Transactions
 Stop the SDK instance from observing transactions. </br>
+**⚠️ Please Note**
+
+> You should call this API only when you want the SDK instance to stop listening for purchase transactions.
+> Usually, you should avoid using this API unless you want to create a new SDK instance or altogether opt out from purchase transaction reporting.
+
```

Comment:
This should be called if you would like to stop the Connector from listening to billing transactions. This removes the listener and stops observing new transactions. An example for using this API is if the app wishes to stop sending data to AppsFlyer due to changes in the user's consent (opt-out from data sharing). Otherwise, there is no reason to call this method. 
If you do decide to use it, it should be called right before calling the Android SDK's [`stop`](https://dev.appsflyer.com/hc/docs/android-sdk-reference-appsflyerlib#stop) API